share/spice/tlds/TLDs.js

```diff
@@ -0,0 +1,43 @@
+(function(env) {
+    "use strict";
+    env.ddg_spice_tlds = function(api_result) {
+        if(!api_result) {
+            return Spice.failed('namecheap');
+        }
+        var item = api_result.ApiResponse.CommandResponse.gettldlist;
+        if(!item) {
+            return Spice.failed('namecheap');
+        }
+        /* extract data from JSON */
+        
+        var tldName = item.tldName;
+        var tldType = item.tldType;
+       
+        var data = {
+            tldType: tldType
+            tldName: tldName
+        };
+        Spice.add({
+            id: 'namecheap',
+            name: 'tldlist',
+            data: data,
```

Comment:
https://duck.co/ia/, checkout the "Topics" dropdown on the right. Those are what should be used for the `name` of IA's. This should help to organize and standardize the Tabs on the SERP.